File: Test Averages.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int a,b,c,x,y,z;
        scanf("%d%d%d",&a,&b,&c);
        x=(a+b)/2;
        y=(b+c)/2;
        z=(a+c)/2;
        if(x>=35&&y>=35&&z>=35)
        printf("PASS\n");
        else
        printf("FAIL\n");
    }
}

File: Test Averages.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int a=sc.nextInt();
            int b=sc.nextInt();
            int c=sc.nextInt();
            if(((a+b)/2)<35||((b+c)/2)<35||((a+c)/2)<35)
            System.out.println("FAIL");
            else
            System.out.println("PASS");
        }
    }
}